resources/views/components/pages/listing/show/floor.blade.php

```diff
@@ -88,8 +88,17 @@
                 </div>
             </div>
         @endif
+        
         <!-- Map -->
         <div class="mt-6 md:mt-12 xl:mt-24">
+            <!-- Address -->
+        @if ($hotel->physical_address)
+            <div class="mt-6 md:mt-12 xl:mt-24">
+                <div class="text-base md:text-lg xl:text-xl 2xl:text-2xl text-gray-500 font-bold">
+                    {{ $hotel->getTranslation('physical_address', app()->getLocale()) }}
```

Comment:
just `$hotel->physical_address` should work since you added it in `$translatable` array. It's resposnsibilty of spatie/trasnslatable package.